uber/site_sections/summary.py

```diff
@@ -225,13 +225,13 @@ def restricted_untaken(self, session):
                     untaken[job.location][hour].append(job)
         flagged = []
         for attendee in attendees:
-            if attendee.trusted and not attendee.is_dept_head:
+            if not attendee.is_dept_head:
                 overlapping = defaultdict(set)
                 for shift in attendee.shifts:
                     if not shift.job.restricted:
                         for dept in attendee.assigned_depts_ints:
                             for hour in shift.job.hours:
-                                if hour in untaken[dept]:
+                                if attendee.trusted_in(dept) and hour in untaken[dept]:
                                     overlapping[shift.job].update(untaken[dept][hour])
```

Comment:
That's fine; in general we don't have any unit tests for our reports, period.  Kind of a shame, but those kinds of tests are typically lower priority than testing the fundamentals of what's in ``models.py`` and places like that.